packages/logo/src/logos/MonochromeLogoMark.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { SVGLogoProps } from '../types';
+
+export default function MonochromeLogoMark({ fill, className }: SVGLogoProps) {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 216.56 448.5"
+      data-testid={`${fill}-monochrome-logomark`}
```

Comment:
We shouldn't render this in our component. If we spread `rest`, we can let the test pass this ID.

Reply:
👍